dotCMS/src/main/java/com/dotcms/visitor/domain/VisitorSerializer.java

```diff
@@ -16,16 +16,27 @@ public void serialize(Visitor visitor, JsonGenerator gen, SerializerProvider ser
 
     if (visitor.getPersona() != null) {
       Persona persona = visitor.getPersona();
-      visitorBuilder.put("persona", ImmutableMap.of(
-          "keyTag", persona.getKeyTag(), 
-          "identifier",persona.getIdentifier(),
-          "tags",persona.getTags(),
-          "title",persona.getTitle()
-          )
-        );
+      ImmutableMap.Builder<String,Object> builder =ImmutableMap.builder();
+      if(persona.getKeyTag() !=null) {
```

Comment:
I think this ifs are not necessary, because if you don't add the entry into the map then the attribute is not add into the json